homeassistant/const.py

```diff
@@ -323,6 +323,7 @@ class Platform(StrEnum):
 STATE_OPENING: Final = "opening"
 STATE_CLOSED: Final = "closed"
 STATE_CLOSING: Final = "closing"
+STATE_BUFFERING: Final = "buffering"
```

Comment:
We may need a new translation string, right?

Reply:
Good point 👍

Reply:
And device automations may need to be updated.

Reply:
And reproduce state?